### DISCUSSION TITLE
Failed to enumerate files in -Path | Cannot bind argument to parameter 'Session' because it is null

### DISCUSSION THREAD

**Discussion by quazimr** [dataplat/dbatools#8023] [Q&A]
While running Update-DbaInstance I am facing following error :
 Failed to enumerate files in -Path | Cannot bind argument to parameter 'Session' because it is null. 
It was working with old dbatools version, but getting above error with latest version.

**Comment by potatoqualitee** [comment]
Thank you for the report! @quazimr can you run it again and then post the output of `Get-DbatoolsError`, then @andreasjordan can take a look. i think he made a recent change.
